Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
You're right, that is better. Seeing it written out w/your explanation I get @serhiy-storchaka 's point about not doubling the literal. I'll tweak that and fix the docstring now.